libs/island-ui/core/src/lib/Main/Main.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { useRef } from 'react'
```

Comment:
Does this main component has to be part of the island-ui? maybe it could be local to the web project?

Reply:
I've moved it to the web project